apps/web/src/design-system/icons/index.ts

```diff
@@ -76,7 +76,7 @@ export { ChevronDown } from './arrows/ChevronDown';
 export { ChevronLeft } from './arrows/ChevronLeft';
 export { ChevronRight } from './arrows/ChevronRight';
 export { ChevronUp } from './arrows/ChevronUp';
-export { DoubleArrowRight as CircleArrowRight } from './arrows/CircleArrowRight';
```

Comment:
DoubleArrowRight still exists. Should we export it anyway?

Reply:
We do further up in this file

Reply:
Here: https://github.com/novuhq/novu/pull/3427/files/20ff5d06eaf1de5b7e2718086579d68f50dc65fc#diff-4b419a3ec6658381b5af0eb97b87038a47f68ca04bdbb81f2d1754c1ed19a5e5L72